label_sync/labels.yaml

```diff
@@ -864,3 +864,19 @@ repos:
         name: area/prow/pod-utilities
         target: both
         addedBy: label
+  kubernetes/community:
+    labels:
+      - color: cc99ff
+        description: Categorizes issue or PR as related to a KEP.
```

Comment:
That said, I don't see mention of the reason or need for `kind/kep` in the meta-kep proposal.  This description is too vague.  ie: what I do not want to see is PR's that implement a KEP being labeled as `kind/kep`, that is not a kind of work.

I was assuming it was solely to be able to query what PR's are KEPs as an interim to transitioning them into kubernetes/keps.  Which now that I think of it, does seem like an appropriate use for an `area/keps` label, to signify something is going to be done with these as part of that effort.